src/index.html

```diff
@@ -7,7 +8,51 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+  <!-- #endregion -->
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star "></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
```

Comment:
```suggestion
    <div class="stars stars--1">
      <div class="stars__star "></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
    
    <div class="stars stars--2">
```

Add empty line between multiple sibling blocks of html

Reply:
is empty line must be between "stars__star" inside "stars"?